teacher.rb

```diff
@@ -0,0 +1,14 @@
+require './person'
+
+class Teacher
```

Comment:
- [ ] `Inherits from Person`. Kindly apply the same Hint and comment I wrote in the student class. Also, you can refer to the project requirements.

Reply:
well resolved ✔️